go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -301,7 +301,7 @@ func (qre *QueryExecutor) Stream(callback StreamCallback) error {
 						return err
 					}
 					defer dbConn.Recycle()
-					return qre.execStreamSQL(dbConn, sql, func(result *sqltypes.Result) error {
+					return qre.execStreamSQL(dbConn, qre.connID != 0, sql, func(result *sqltypes.Result) error {
```

Comment:
If you don't feel strongly about this .. i would rather keep it like this so in the future if anyone do any kind of refactoring/change in this code block, it will be less likely to result in regression.